sprints/dashboard/helpers.py

```diff
@@ -0,0 +1,366 @@
+            self.story_points = 0
+        self.reviewer_1: JiraUser = getattr(issue.fields, issue_fields['Reviewer 1'], None)
+        # We don't want to treat commitments from the other cell as "Unassigned".
+        if self.reviewer_1 and self.reviewer_1.name not in cell_members:
+            self.reviewer_1 = "Other Cell"
+
+        self.review_time = self.calculate_review_time()
+        self.assignee_time = max(self.time_estimate - self.review_time, 0)  # We don't want negative values here
+
+    def get_bot_directive(self, pattern) -> int:
+        """Retrieves special directives placed for the Jira bot."""
+        try:
+            return int(re.search(pattern, self.description).group(1))
+        except (IndexError, AttributeError, TypeError):
+            return 0
+
+    def calculate_review_time(self) -> int:
+        """
+        Calculate time needed for the review.
+        Unless directly specified (with Jira bot directive), we're planning 2 hours for stories bigger than 3 points.
+        """
+        planned = self.get_bot_directive(SPRINT_REVIEW_DIRECTIVE)
+        if planned:
+            return planned * SECONDS_IN_HOUR
+
+        if self.story_points <= 3:
+            return SECONDS_IN_HOUR
+        return 2 * SECONDS_IN_HOUR
+
+    def get_recurring_time(self) -> int:
+        """Get required assignee time for the recurring story."""
+        planned = self.get_bot_directive(SPRINT_RECURRING_DIRECTIVE)
+        return planned * SECONDS_IN_HOUR
+
+    def get_epic_management_time(self) -> int:
+        """Get required assignee time for managing the epic."""
+        planned = self.get_bot_directive(SPRINT_EPIC_DIRECTIVE)
+        return planned * SECONDS_IN_HOUR or SPRINT_HOURS_RESERVED_FOR_EPIC_MANAGEMENT * SECONDS_IN_HOUR
+
+
+class DashboardRow:
+    """Represents single dashboard row (user)."""
+
+    def __init__(self, user: Union[JiraUser, str, None]) -> None:
+        super().__init__()
+        self.user = user
+        try:
+            self.local_user = User.objects.get(email=self.user.emailAddress)
+        except (User.DoesNotExist, AttributeError):
+            self.local_user = None
+        self.current_remaining_assignee_time = 0
+        self.current_remaining_review_time = 0
+        self.current_remaining_upstream_time = 0
+        self.future_remaining_assignee_time = 0
+        self.future_remaining_review_time = 0
+        self.future_epic_management_time = 0
+        self.goal_time = 0
+        self.current_invalid = []
+        self.future_invalid = []
+
+    def set_goal_time(self, goal) -> None:
+        """
+        Calculate goal time for this user.
+        Use default if time not specified in user's profile.
+        """
+        self.goal_time = goal - SPRINT_HOURS_RESERVED_FOR_MEETINGS * SECONDS_IN_HOUR
+
+    @property
+    def committed_time(self) -> float:
+        """Calculate summary time for the upcoming sprint."""
+        return self.current_remaining_assignee_time \
```

Comment:
Use parenthesis to avoid `\`
```suggestion
        return (self.current_remaining_assignee_time
                + self.current_remaining_review_time
               + self.current_remaining_upstream_time
               + self.future_remaining_assignee_time
               + self.future_remaining_review_time
               + self.future_epic_management_time)
```